From 17fb168f007eb4200a1d3325898cf5256a029d81 Mon Sep 17 00:00:00 2001
From: Phil Sutter <phil@nwl.cc>
Date: Sat, 5 Oct 2024 14:13:44 +0200
Subject: nft: Make add_log() static

It is not used outside of nft.c, though in the wrong position so keep
the declaration but right above its caller.

Signed-off-by: Phil Sutter <phil@nwl.cc>
---
 iptables/nft.c | 4 +++-
 iptables/nft.h | 1 -
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/iptables/nft.c b/iptables/nft.c
index 21a7e211..d1e04d2b 100644
--- a/iptables/nft.c
+++ b/iptables/nft.c
@@ -1595,6 +1595,8 @@ int add_verdict(struct nftnl_rule *r, int verdict)
 	return 0;
 }
 
+static int add_log(struct nftnl_rule *r, struct iptables_command_state *cs);
+
 int add_action(struct nftnl_rule *r, struct iptables_command_state *cs,
 	       bool goto_set)
 {
@@ -1623,7 +1625,7 @@ int add_action(struct nftnl_rule *r, struct iptables_command_state *cs,
 	return ret;
 }
 
-int add_log(struct nftnl_rule *r, struct iptables_command_state *cs)
+static int add_log(struct nftnl_rule *r, struct iptables_command_state *cs)
 {
 	struct nftnl_expr *expr;
 	struct xt_nflog_info *info = (struct xt_nflog_info *)cs->target->t->data;
diff --git a/iptables/nft.h b/iptables/nft.h
index 8f17f310..09b4341f 100644
--- a/iptables/nft.h
+++ b/iptables/nft.h
@@ -197,7 +197,6 @@ int add_match(struct nft_handle *h, struct nft_rule_ctx *ctx,
 int add_target(struct nftnl_rule *r, struct xt_entry_target *t);
 int add_jumpto(struct nftnl_rule *r, const char *name, int verdict);
 int add_action(struct nftnl_rule *r, struct iptables_command_state *cs, bool goto_set);
-int add_log(struct nftnl_rule *r, struct iptables_command_state *cs);
 char *get_comment(const void *data, uint32_t data_len);
 
 enum nft_rule_print {
-- 
cgit v1.2.3

